app/src/test/java/comcmput301f16t01/github/carrier/UserProfileTests.java

```diff
@@ -46,21 +46,21 @@ public void uniqueDriverUsername() {
      */
     @Test
     public void editRiderInfo() {
-        Rider riderOne = new Rider("start username");
-        riderOne.setEmail("start email");
-        riderOne.setPhone("start phone");
+        UserController uc = new UserController();
+        uc.getLoggedInUser().setEmail("start email");
```

Comment:
Error proofing this test (not necessary for merge): Use a String variable instead of setting it in the initialization. This makes testing easier if we happen to change the string here and forget to change it in the assertion.